catalog/catalog-app/src/main/resources/features.xml

```diff
@@ -345,6 +345,20 @@
         <feature>catalog-plugin-metacard-validation</feature>
     </feature>
 
+    <feature name="catalog-core-validationparser" install="manual" version="${project.version}"
```

Comment:
Any reason we wouldn't want this (and the core-validator feature) installed by default?